include/tvm/runtime/vm/executable.h

```diff
@@ -144,6 +144,13 @@ class Executable : public ModuleNode {
    */
   std::string GetVirtualDevices() const;
 
+  /*!
```

Comment:
Can you clarify what exactly the return value is here given its just a `std::string`.